runengine.py

```diff
@@ -0,0 +1,21 @@
+from agent_class import agent
+from tictactoe_engine import tictactoeEngine
+import json
+from tictactoe_engine import Code
+
+
+# def test1():
+moves1 = [1, 2, 6, 4, 9]
+moves2 = [7, 5, 3, 8]
+agent1 = agent("anik", moves1, 1)
+agent1_ready = agent1.agent_instance()
+agent2 = agent("arka", moves2, -1)
+agent2_ready = agent2.agent_instance()
+engine_input = tictactoeEngine(agent1_ready[0], agent1_ready[1], agent1_ready[2], agent2_ready[0], agent2_ready[1], agent2_ready[2] ) 
```

Comment:
Class variables can be directly accessed by their objects. (examples: `agent1.name`, `agent2.moves` etc)
Functions like `agent_instance()` are not required to access the member variables.